windows.md

```diff
@@ -483,7 +483,7 @@ With those compatibility things out of the way, you're ready to start the system
     Copy each line in the following text, paste it in Hyper and hit return.
 
     ```bash
-    choco install androidstudio
+    choco install androidstudio -f
```

Comment:
Seems like some packages are missing the `--yes` flag, let's fix that in this PR too

Reply:
ac2b6b4326d4cf5f5d372d1b920fe2c0e902f263